src/app/zap-templates/zcl/data-model/chip/camera-av-settings-user-level-management-cluster.xml

```diff
@@ -86,22 +87,22 @@ Git: 0.7-summer-2025-ncr-1615-gf52f22908
         <feature name="MZOOM"/>
       </mandatoryConform>
     </attribute>
-    <attribute code="0x0005" side="server" name="TiltMin" define="TILT_MIN" type="int16s" min="-180" max="179" optional="true">
+    <attribute code="0x0005" side="server" name="TiltMin" define="TILT_MIN" type="int16s" min="-180" max="0" optional="true">
       <mandatoryConform>
         <feature name="MTILT"/>
       </mandatoryConform>
     </attribute>
-    <attribute code="0x0006" side="server" name="TiltMax" define="TILT_MAX" type="int16s" min="-179" max="180" optional="true">
+    <attribute code="0x0006" side="server" name="TiltMax" define="TILT_MAX" type="int16s" min="1" max="180" optional="true">
```

Comment:
I have a PR in work that handles this and updates the server logic and the test scripts. Can you remove this from this PR so that it's only handling the removal of the provisional marker. 

Sams for TitlMin and PanMin and PanMax